libsodium-bindings/src/LibSodium/Bindings/AEAD.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE CApiFFI #-}
+
+module LibSodium.Bindings.AEAD where
+
+import Foreign.C.Types (CSize(..), CUChar(..), CULLong(..), CInt(..))
+import Foreign.Ptr (Ptr)
+
+-- Bindings to AEAD constructions, specifically XChaCha20.
+
+
+-- | This function encrypts a message, and then appends the authentication tag
+-- to the encrypted message.
```

Comment:
Two things are missing:

* The `@since 0.0.1.0` annotation
* The reference to the libsodium manual, which takes the form of:

```haskell
-- /See:/ [<name of the C function](link to the anchor of the section of the manual)
```

You can generalise this comment to the rest of the functions